xcm/update_to_prod.py

```diff
@@ -0,0 +1,134 @@
+"""
+This script compares the dev and prod transfers.json files, finds the difference and asks which changes to apply, then updates the prod file.
+"""
+
+import json
+
+def ask_to_update():
+    update = input("Would you like to update that data? (y/n)")
+    if update.lower() == 'y':
+        return True
+    return False
+
+def update_networks(dev, prod, meta_dict):
+    for key, value in dev.items():
```

Comment:
I suggest to give more specific names to iterating variables - currently it is not quite clear what are `key` and `value` here

For example:

`for chainId, chain in dev_chains`

Reply:
thank you, fixed